### PR TITLE
Post restore config fix

### PR DESCRIPTION
**Pull request by ericsyh**
### What problem does this PR solve? 
The description of level-1-compact, compact has some mistakes in lightning.toml. 

### What is changed and how it works?
Fix the mistake in lightning.toml and update the default value of `compact` config.  

### Check List 

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### CODE REVIEW

File: tidb-lightning.toml [hunk 1]
```diff
@@ -138,10 +138,11 @@ checksum-table-concurrency = 16
 # if set true, checksum will do ADMIN CHECKSUM TABLE <table> for each table.
 checksum = true
 # if set to true, compact will do level 1 compaction to tikv data.
-# if this setting is missing, the default value is also true.
+# if this setting is missing, the default value is also false.
```
Comment (kennytm) [inline tidb-lightning.toml]: Please remove the "also"

File: tidb-lightning.toml [hunk 2]
```diff
@@ -138,10 +138,11 @@ checksum-table-concurrency = 16
...
 level-1-compact = false
 # if set true, compact will do full compaction to tikv data.
-compact = true
+# if this setting is missing, the default value is also false.
```
Comment (kennytm) [inline tidb-lightning.toml]: Ditto

File: tidb-lightning.toml (hunk 1 above)
Comment (ericsyh) [reply tidb-lightning.toml]: Removed in https://github.com/pingcap/tidb-lightning/pull/202/commits/d7b952af22deacdd093c01a497f77fa03c146edb

File: tidb-lightning.toml (hunk 2 above)
Comment (ericsyh) [reply tidb-lightning.toml]: Removed in https://github.com/pingcap/tidb-lightning/pull/202/commits/d7b952af22deacdd093c01a497f77fa03c146edb

**Pull request merged**